lib/sales_analyst.rb

```diff
@@ -1,94 +1,72 @@
 require 'pry'
-
+require_relative './sales_engine'
+require_relative './mathable'
 class SalesAnalyst
+  include Mathable
   attr_reader :sales_engine
   def initialize(sales_engine)
     @sales_engine = sales_engine
   end
-
+  def total_items
+    sales_engine.count_items
+  end
+  def total_merchants
+    sales_engine.count_merchants
+  end
+  def average_item_price
+    sales_engine.average_item_price
+  end
   def average_items_per_merchant
-    (total_items.to_f / total_merchants).round(2)
+    average_stuff(total_items.to_f, total_merchants)
+  end
+  def items_per_merchant
+    sales_engine.items
+  end
+  def average_item_price
+    sales_engine.average_item_price
+  end
+  def item_price_standard_deviation
+    average = average_item_price
+    total = sales_engine.pass_item_array.sum do |item|
+      (average - item.unit_price_to_dollars)**2
```

Comment:
Looks good. Might be able to break out into math module

Reply:
Good idea